react-app/src/library/services/vendor-implementations/plantronics/plantronics.ts

```diff
@@ -62,40 +92,49 @@ export default class PlantronicsService extends VendorImplementation {
   }
 
   async pollForCallEvents() {
-    if (this.isConnected && this.isActive && !this.disableEventPolling) {
+    if (this.callEventsTimerId) {
+      clearTimeout(this.callEventsTimerId);
+      this.callEventsTimerId = null;
+    }
+
+    if(this.isConnected && this.isActive && !this.disableEventPolling) {
       await this.getCallEvents();
     }
-    setTimeout(() => {
+
+    this.callEventsTimerId = setTimeout(() => {
       console.log('**** POLLING FOR CALL EVENTS ****');
       this.pollForCallEvents();
     }, this.activePollingInterval);
   }
 
   async pollForDeviceStatus() {
+    if (this.deviceStatusTimerId) {
+      clearTimeout(this.deviceStatusTimerId);
+      this.deviceStatusTimerId = null;
+    }
+
     if (this.isConnected && !this.isConnecting && !this.disableEventPolling) {
       await this.getDeviceStatus();
     }
-    setTimeout(
-      () => {
+
+    this.deviceStatusTimerId = setTimeout(() => {
         console.log('**** POLLING FOR DEVICE STATUS ****');
         this.pollForDeviceStatus();
```

Comment:
I mentioned this on your last PR, just mentioning it again on this one. Chrome _may_ throttle these. I think they throttle up to a minute would could cause some issues with the 2 sec disconnected device delay. If there is an active device, we should be ok since (hopefully) the audio stream will keep use from getting throttled.